libcaf_core/test/typed_actor_view.cpp

```diff
@@ -0,0 +1,183 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#define CAF_SUITE typed_actor_view
```

Comment:
Is there anything else to test for the `typed_actor_view` aside from stream / flow-releated functionality?